filter/usr/share/nethesis/NethServer/Template/Mail/Filter.php

```diff
@@ -23,24 +23,26 @@
 
 $spamCheckbox = $view->fieldsetSwitch('SpamCheckStatus', 'enabled', $view::FIELDSETSWITCH_CHECKBOX | $view::FIELDSETSWITCH_EXPANDABLE)
     ->setAttribute('uncheckedValue', 'disabled')
-    ->insert($view->slider('SpamGreyLevel', $view::LABEL_ABOVE)
+    ->insert($view->slider('SpamGreyLevel', $view::LABEL_ABOVE | $view::STATE_DISABLED | $view::STATE_READONLY)
         ->setAttribute('min', $view->getModule()->spamThresholdMin)
         ->setAttribute('max', $view->getModule()->spamThresholdMax)
         ->setAttribute('step', 0.1)
         ->setAttribute('label', $T('SpamGreyLevel ${0}'))
     )
-    ->insert($view->slider('SpamTag2Level', $view::LABEL_ABOVE)
+    ->insert($view->slider('SpamTag2Level', $view::LABEL_ABOVE | $view::STATE_DISABLED | $view::STATE_READONLY)
         ->setAttribute('min', $view->getModule()->spamThresholdMin)
         ->setAttribute('max', $view->getModule()->spamThresholdMax)
         ->setAttribute('step', 0.1)
         ->setAttribute('label', $T('SpamTag2Level ${0}'))
     )
-    ->insert($view->slider('SpamKillLevel', $view::LABEL_ABOVE)
+    ->insert($view->slider('SpamKillLevel', $view::LABEL_ABOVE | $view::STATE_DISABLED | $view::STATE_READONLY)
         ->setAttribute('min', $view->getModule()->spamThresholdMin)
         ->setAttribute('max', $view->getModule()->spamThresholdMax)
         ->setAttribute('step', 0.1)
         ->setAttribute('label', $T('SpamKillLevel ${0}'))
     )
+    ->insert($view->literal($view->translate(ThresholdMoved2Rspamd).'<br/><br/>')
```

Comment:
When `literal()` is used remeber to escape HTML special characters with `htmlspecialchars()`